starlingx/inventory/v1/interfaceDataNetworks/results.go

```diff
@@ -50,11 +50,11 @@ type InterfaceDataNetwork struct {
 
 	// DataNetworkUUID defines the system assigned unique UUID of the associated
 	// data network.
-	DataNetworkUUID string `json:"network_uuid"`
+	DataNetworkUUID string `json:"datanetwork_uuid"`
```

Comment:
Same question... was the mapping wrong?

Reply:
The API and system --debug has it as datanetwork_uuid (https://docs.starlingx.io/api-ref/config/api-ref-sysinv-v1-config.html?expanded=shows-information-about-a-specific-interface-data-network-detail#interface-data-networks)